src/game/WorldHandlers/CharacterHandler.cpp

```diff
@@ -135,10 +156,87 @@ class CharacterHandler
                 delete holder;
                 return;
             }
+#ifdef ENABLE_PLAYERBOTS
+            ObjectGuid guid = ((LoginQueryHolder*)holder)->GetGuid();
+#endif
             session->HandlePlayerLogin((LoginQueryHolder*)holder);
```

Comment:
Is it really necessary to have one line of code in between two of the same ifdefs? Maybe move this line above the ifdef and join the two ifdefs together into one.

Reply:
Seems holder is deleted by HandlePlayerLogin. So this is necessary and doesnt need any changes.